MiniGamesBox Classic/pom.xml

```diff
@@ -126,7 +126,7 @@
         <dependency>
             <groupId>plugily.projects</groupId>
             <artifactId>MiniGamesBox-Inventory</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
wouldn't return a version yet?

Reply:
It's easier because you can just update the parent version and it'll apply to all modules.